controller/workitem.go

```diff
@@ -605,6 +611,26 @@ func ConvertWorkItem(request *goa.RequestData, wi workitem.WorkItem, additional
 	return op
 }
 
+// WorkItemIncludeHasChildren adds meta information about existing children
+func WorkItemIncludeHasChildren(appl application.Application, ctx context.Context) WorkItemConvertFunc {
```

Comment:
No it is not necessary and has been fixed here: 04067938d722ab9db547082d8db44ca2ab2354cc